src/picongpu/include/simulation_defines/param/particleMerging.param

```diff
@@ -0,0 +1,34 @@
+/* Copyright 2017 Heiko Burau
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+namespace picongpu
+{
+namespace plugins
+{
+namespace particleMerging
+{
+
+	/* maximum number of active Voronoi cells per supercell */
```

Comment:
- need `/**`
- Can I choose any number or is the number of cells restricted by the size of the shared memory?